src/oc_stat_view.erl

```diff
@@ -1,3 +1,9 @@
+%%%------------------------------------------------------------------------
```

Comment:
Can we have a test for this? Here is what we added in Java:
https://github.com/census-instrumentation/opencensus-java/blob/master/buildscripts/checkstyle.xml#L25

Reply:
hmm cool idea, do you think it should go to this PR?

Reply:
Does elvis have a check for that?

Reply:
Not necessary in this PR but good to have it at one point.

Reply:
@tsloughter looks like it doesn't have it https://github.com/inaka/elvis_core/wiki/Rules